pytorch_lightning/utilities/apply_func.py

```diff
@@ -35,6 +35,9 @@
     Batch = type(None)
 
 
+CPU_DEVICES = ("cpu", torch.device("cpu"))
```

Comment:
This can be protected

```suggestion
_CPU_DEVICES = ("cpu", torch.device("cpu"))
```